ext/standard/basic_functions.stub.php

```diff
@@ -3359,29 +3410,56 @@ function socket_set_timeout($stream, int $seconds, int $microseconds = 0): bool
 
 /* type.c */
 
+/**
+ * @compile-time-eval
+ */
 function gettype(mixed $value): string {}
 
+/**
+ * @compile-time-eval
+ */
 function get_debug_type(mixed $value): string {}
 
 function settype(mixed &$var, string $type): bool {}
 
+/**
+ * @compile-time-eval
+ */
 function intval(mixed $value, int $base = 10): int {}
 
+/**
+ * @compile-time-eval
+ */
 function floatval(mixed $value): float {}
 
 /** @alias floatval */
 function doubleval(mixed $value): float {}
```

Comment:
I'm not sure, do we need to specify this for aliases too to work?

Reply:
AFAK, and from already present functions, aliases does not need any flags duplicated manually

but someone else should confirm

Reply:
Can confirm, you don't need to specify it on aliases.